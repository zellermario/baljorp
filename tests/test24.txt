create SJ mezo 7

create Eszkimo eszkimo mezo
create TorekenyAso ta player eszkimo

useobject ta eszkimo
useobject ta eszkimo
useobject ta eszkimo
pass eszkimo
