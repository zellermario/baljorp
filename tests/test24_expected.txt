Fields:
SJ mezo (1) - objects: ?
	neighbours: []

Buildings:

Actors:
> Eszkimo eszkimo (heat: 5) on mezo
  Inventory: []
